internal/lookout/ui/src/services/ComputeResourcesService.ts

```diff
@@ -0,0 +1,111 @@
+import { load } from "js-yaml"
+
+// A simple interface that our PodSpec has.
+export interface JobYaml {
+  podSpec: {
+    containers: Container[]
+  }
+}
+
+export interface ContainerInfo {
+  command?: string[]
+  arguments?: string[]
+  cpu?: string[]
```

Comment:
Why does a single `ContainerInfo` have a list of values for cpu, memory, gpu? Each container info should only have a single value. Don't think this model makes sense here, I'd probably use the `Container` interface below, and have a list of them.

Reply:
Done.